edx_proctoring/api.py

```diff
@@ -1375,12 +1380,17 @@ def get_attempt_status_summary(user_id, course_id, content_id):
     # practice exams always has an attempt status regardless of
     # eligibility
     if credit_service and not exam['is_practice_exam']:
-        credit_state = credit_service.get_credit_state(user_id, unicode(course_id))
+        credit_state = credit_service.get_credit_state(user_id, unicode(course_id), return_course_info=True)
         if not _check_eligibility_of_enrollment_mode(credit_state):
             return None
 
     attempt = get_exam_attempt(exam['id'], user_id)
-    status = attempt['status'] if attempt else ProctoredExamStudentAttemptStatus.eligible
+    if attempt:
+        status = attempt['status']
+    elif not exam['is_practice_exam'] and has_due_date_passed(credit_state.get('course_end_date', None)):
```

Comment:
This case is not covered by the ddt addition below. I think we need to add a test for when there is no attempt, it is not a practice exam, and the course_end_date has passed.